### ISSUE TITLE
Pass filter name to lambda defaults

### ISSUE THREAD

**Issue by dush** [orgsync/active_interaction#357]
It would be nice to pass hash of name and options of particullar filter to filter default. 
Then should be easy to initialize default values of bunch of inputs from provided object (model) like this

```
object :account
with_options default: ->(name:, **options) { account.send(name) } do
  string :first_name
  string :last_name
end
```

**Comment by tfausak** [comment]
I like this idea. I would prefer to pass the entire filter into the default instead of the name and options. The only catch is that the `default` option would have to support lambdas with either arity 0 (the old way) or 1 (this way).

**Comment by tfausak** [comment]
I made a proof of concept in #358. Does it work for you?

**Comment by dush** [comment]
@tfausak great. I've  tried it in our application interaction where we neeed to initialize bunch of 15+ inputs from model attributes and it works.

**Comment by tfausak** [comment]
Released in version 3.1.0. Thanks for the suggestion!
